microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
 keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides
 ---
 
-# Microsoft Edge User Agent String (Desktop)  
+# Retrieving User Agent Data in Microsoft Edge 
 
-A user agent \(UA\) string is able to be used to detect what version of a specific browser is being used on a certain operating system.  Like other browsers, Microsoft Edge includes this information in the `User-Agent` HTTP header whenever it makes a request to a site.  It may also be accessed via JavaScript by querying the value of `navigator.userAgent`.  
+The User Agent \(UA\) can be used detect what version of a specific browser is being used on a certain operating system. Microsoft Edge offers two mechanisms for retrieving User Agent information: via User Agent Client Hints (recommended) and the legacy User Agent string. These can be accessed as follows: 
+
+**User Agent Client Hints (Recommended)**
+-	*Server-side*: UA Client Hints are sent by default with the `Sec-CH-UA` `HTTPS` header 
+-	*Client-side*: UA Client Hints can be accessed via JavaScript by querying the value of `navigator.userAgentData`
+
+**User Agent String**
+-	*Server-side*: UA strings are sent by default with the User-Agent `HTTP` header 
+-	*Client-side*: UA strings can be accessed via JavaScript by querying the value of `navigator.userAgent`
 
 Microsoft recommends that web developers make use of [feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) whenever possible to improve code maintainability, reduce code fragility, and eliminate the risk of code breakage in the event of future UA string updates.  
 
 For cases where feature detection is not applicable and UA detection must be used, the format of the Microsoft Edge UA on desktop is as follows:
 
+## User Agent Client Hints
+Starting from Edge90 Stable, developers can access browser information in a cleaner, more privacy preserving way. 

```

Comment:
We should have a space between "Edge" and the version number. "Edge 90"

Reply:
Updated.